NOTICE

```diff
@@ -3,3 +3,7 @@ Copyright [2014-2016] The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
+
+
+Portions of this software were developed at
```

Comment:
Right, I'm not saying that's a bad thing.  My concern is this page: http://www.apache.org/dev/licensing-howto.html#mod-notice, which includes

    Do not add anything to NOTICE which is not legally required.